src/assets/sass/protocol/components/_breadcrumb.scss

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-c-breadcrumb {
+    background-color: #f3f3f3;
+    margin-bottom: 0;
+    padding: 8px 18px;
+
+    .mzp-c-breadcrumb-item {
+        display: inline;
+        font-weight: 400;
+
+        .mzp-c-breadcrumb-item-link {
+            text-decoration: none;
+        }
+
+        &:not(.mzp-c-breadcrumb-title) {
```

Comment:
Support for `:not` is really recent, we should probably find a more backwards-compatible method.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:not seems like its the `selector list` that isn't well supported (`:not(div, .my-class)`) when its just one selector it seems like its supported back to IE 9. Should we still find something else to use?

Reply:
Ah, right you are. When I checked caniuse I was looking at the entry for multi-selector, but we should be fine with a single selector. My mistake 👍